### ISSUE TITLE
Enhancement: Credit abandoments as kills when vehicle abandonment (unnoficial) is turned on

### ISSUE THREAD

**Issue by Tamren** [MegaMek/mekhq#1287]
If an enemy vehicle takes weapon damage that immobilizes it or destroys the engine and the crew abandons the vehicle as a result, the unit that caused the immobilization should get the kill credit.

**Comment by NickAragua** [comment]
Yeah, this bugs me on occasion when I come up against a half a battalion of Vedettes or something. No ETA, but a good idea.

**Comment by binaryspica** [comment]
Yeah, we need this desperately. It is too annoying to manually give them a brand new unit and hiring a new person and make the person prisoner(if we have captured any of).
